### DISCUSSION TITLE
Provider config file format

### DISCUSSION THREAD

**Discussion by meetgyn** [projectdiscovery/subfinder#426] [Q&A] [answered]
How to make the correct configuration setting.yaml? 

![image](https://user-images.githubusercontent.com/3695608/121285061-20822280-c8b4-11eb-843a-671dae1dc2df.png)
Should I put the API only in these lines? Within [ ] or out?

**Comment by ehsandeep** [comment]
@meetgyn checkout the example section at https://github.com/projectdiscovery/subfinder#post-installation-instructions

**Reply by meetgyn** [reply]
> 
> 
> @meetgyn checkout the example section at https://github.com/projectdiscovery/subfinder#post-installation-instructions

Thks  man.

**Comment by salooali** [comment, marked as answer]
@meetgyn
 No, you can write in the following format:
remove the "[ ]" go to the next line by pressing the Enter key then use 4 spaces most probably put the "-" symbol followed 
by a space " " then you can paste your API key. 

for censys you need to have an API-id and secret key. The format will be like this. 
api-id:secret-key as shown in an example.

Examples:
binaryedge: 
    - 87a8938c************1f192
censys: 
    - b3892a82-4******************347a845:XvdEIFE*************OPM7v1Dr2
certspotter: 
    - 40690_wYt******************ZMwi
chaos: []
dnsdb: []
github: 
    - ghp**************************2aaUwj
intelx: 
    - 14537d***************************37ec58c60b77
passivetotal: 
    - mail@gmail.com:api-key
